core/src/test/java/com/valtech/aapm/restrictions/PropertyValueRestrictionProviderTest.java

```diff
@@ -74,20 +70,19 @@ public void getPattern_nominal_case() {
         HasPropertyValuesPattern expectedRestrictionPattern =
                 new HasPropertyValuesPattern(matchedConditionPropertyValue, oakPath);
 
-
         RestrictionPattern computedRestrictionPattern = testedProvider.getPattern(oakPath, restrictions);
 
         assertTrue(computedRestrictionPattern instanceof HasPropertyValuesPattern);
         assertEquals(expectedRestrictionPattern, computedRestrictionPattern);
     }
 
     @Test
-    public void getPattern_returns_empty_pattern_restriction_when_special_property_is_a_multiple_one() {
+    void getPattern_returns_empty_pattern_restriction_when_special_property_is_a_multiple_one() {
         Type<Iterable<String>> type = Type.STRINGS;
         String oakPath = "/my/path";
         String name = "hasPropertyValues";
-        String value1 = "deny#string$cq:tags==properties:orientation/portrait";
-        String value2 = "allow#string$cq:tags==properties:orientation/landscape";
+        String value1 = "deny#string$cq:tagsDOUBLE_EQUALSproperties:orientation/portrait";
```

Comment:
The readability of "deny#string$cq:tagsDOUBLE_EQUALSproperties:orientation/portrait" is not good.
How about adding a space before and after the operator?
Like "deny#string$cq:tags DOUBLE_EQUALS properties:orientation/portrait"

Reply:
having space in start and end is creating more problem therefore I added _